src/bridge/cockpitpcpmetrics.c

```diff
@@ -177,7 +177,7 @@ build_meta (CockpitPcpMetrics *self,
               rc = pmNameInDom (self->metrics[i].desc.indom, vs->vlist[j].inst, &instance);
               if (rc != 0)
                 {
-                  g_warning ("%s: instance name lookup failed: %s", self->name, pmErrStr (rc));
+                  g_warning ("%s: instance name lookup failed: %s.%d: %s", self->name, self->metrics[i].name, vs->vlist[j].inst, pmErrStr (rc));
```

Comment:
I don't see a reason to not just keep this. Can you please just clean up the commit message?

Reply:
Done.